`const_residue` doesn't seem to work outside of crate

(Forgive me if this is an obvious problem based on my lack of Rust experience).

I'm trying to use the modular exp support recently added, but I'm running in to a problem with const_residue. Specifically, I do this:

```rust
impl_modulus!(PMod, U1536, P_STR);
```
where `P_STR` contains the hex string for the modulus. Then, in a test, I do:

```rust
let base = const_residue!(G1, PMod);
```

where `G1` is a big int of the same size as the modulus. However, this leads to a failure saying "module `uint` is private". Specifically:

```
error[E0603]: module `uint` is private
   --> /home/amnesia/Persistent/.cargo/registry/src/github.com-1ecc6299db9ec823/crypto-bigint-0.5.0/src/uint/modular/constant_mod/macros.rs:48:17
    |
48  |         $crate::uint::modular::constant_mod::Residue::<$modulus, { $modulus::LIMBS }>::new(
    |                 ^^^^ private module
    |
note: the module `uint` is defined here
   --> /home/amnesia/Persistent/.cargo/registry/src/github.com-1ecc6299db9ec823/crypto-bigint-0.5.0/src/lib.rs:167:1
    |
167 | mod uint;
    | ^^^^^^^^
```

--- a/src/uint/modular/constant_mod/macros.rs
+++ b/src/uint/modular/constant_mod/macros.rs
@@ -45,7 +45,7 @@ macro_rules! impl_modulus {
 /// For example, `residue!(U256::from(105u64), MyModulus);` creates a `Residue` for 105 mod `MyModulus`.
 macro_rules! const_residue {
     ($variable:ident, $modulus:ident) => {
-        $crate::uint::modular::constant_mod::Residue::<$modulus, { $modulus::LIMBS }>::new(
+        $crate::modular::constant_mod::Residue::<$modulus, { $modulus::LIMBS }>::new(
             &$variable,
         )
     };

--- a/src/uint/modular/constant_mod/macros.rs
+++ b/src/uint/modular/constant_mod/macros.rs
@@ -45,8 +45,6 @@ macro_rules! impl_modulus {
 /// For example, `residue!(U256::from(105u64), MyModulus);` creates a `Residue` for 105 mod `MyModulus`.
 macro_rules! const_residue {
     ($variable:ident, $modulus:ident) => {
-        $crate::modular::constant_mod::Residue::<$modulus, { $modulus::LIMBS }>::new(
-            &$variable,
-        )
+        $crate::modular::constant_mod::Residue::<$modulus, { $modulus::LIMBS }>::new(&$variable)
     };
 }

--- a/tests/const_residue.rs
+++ b/tests/const_residue.rs
@@ -1,0 +1,10 @@
+//! Test to ensure that `const_residue!` works from outside this crate.
+
+use crypto_bigint::{impl_modulus, const_residue, U64, modular::constant_mod::ResidueParams};
+
+impl_modulus!(TestMod, U64, "30e4b8f030ab42f3");
+
+fn _test_fun() {
+    let base = U64::from(2u64);
+    let _base_mod = const_residue!(base, TestMod);
+}

--- a/tests/const_residue.rs
+++ b/tests/const_residue.rs
@@ -1,6 +1,6 @@
 //! Test to ensure that `const_residue!` works from outside this crate.
 
-use crypto_bigint::{impl_modulus, const_residue, U64, modular::constant_mod::ResidueParams};
+use crypto_bigint::{const_residue, impl_modulus, modular::constant_mod::ResidueParams, U64};
 
 impl_modulus!(TestMod, U64, "30e4b8f030ab42f3");
 

